src/microbe_stage/SpawnSystem.cs

```diff
@@ -90,9 +90,9 @@ public SpawnSystem(Node root)
     ///   Adds an externally spawned entity to be despawned
     /// </summary>
     public static void AddEntityToTrack(ISpawned entity,
-        float radius = Constants.MICROBE_SPAWN_RADIUS)
+        float radius = Constants.MICROBE_SPAWN_RADIUS + Constants.DESPAWN_RADIUS_OFFSET)
     {
-        entity.DespawnRadiusSquared = (int)(radius * radius) + Constants.DESPAWN_RADIUS_OFFSET_SQUARED;
+        entity.DespawnRadiusSquared = (int)(radius * radius);
```

Comment:
Wouldn't a power function be a bit clearer there? (and some other places around)

Reply:
Perhaps, I just used what was already there. I can change that if wanted.